src/mcts/search.h

```diff
@@ -204,12 +208,56 @@ class Search {
 // within one thread, have to split into stages.
 class SearchWorker {
  public:
-  SearchWorker(Search* search, const SearchParams& params)
+  SearchWorker(Search* search, const SearchParams& params, int id)
       : search_(search),
         history_(search_->played_history_),
         params_(params),
         moves_left_support_(search_->network_->GetCapabilities().moves_left !=
-                            pblczero::NetworkFormat::MOVES_LEFT_NONE) {}
+                            pblczero::NetworkFormat::MOVES_LEFT_NONE) {
+#if defined(_WIN64) && _WIN32_WINNT >= 0x0601
```

Comment:
Probably should extract this code somewhere else, since it'll be useful for backend threads as well.